plugins/ledger/ledger.py

```diff
@@ -118,12 +123,12 @@ def supports_native_segwit(self):
     def perform_hw1_preflight(self):
         try:
             firmwareInfo = self.dongleObject.getFirmwareVersion()
-            firmware = firmwareInfo['version'].split(".")
-            self.multiOutputSupported = int(firmware[0]) >= 1 and int(firmware[1]) >= 1 and int(firmware[2]) >= 4
-            self.segwitSupported = (int(firmware[0]) >= 1 and int(firmware[1]) >= 1 and int(firmware[2]) >= 10) or (firmwareInfo['specialVersion'] == 0x20 and int(firmware[0]) == 1 and int(firmware[1]) == 0 and int(firmware[2]) >= 4)
-            self.nativeSegwitSupported = int(firmware[0]) >= 1 and int(firmware[1]) >= 1 and int(firmware[2]) >= 10
+            firmware = firmwareInfo['version']
+            self.multiOutputSupported = self.versiontuple(firmware) >= self.versiontuple(MULTI_OUTPUT_SUPPORT)
+            self.nativeSegwitSupported = self.versiontuple(firmware) >= self.versiontuple(SEGWIT_SUPPORT)
+            self.segwitSupported = self.nativeSegwitSupported or (firmwareInfo['specialVersion'] == 0x20 and self.multiOutputSupported)
 
-            if not checkFirmware(firmware):
+            if not checkFirmware(firmwareInfo):
```

Comment:
Why? Looks like this was `firmwareInfo['version'].split(".")`.
Though it seems that function just returns True anyway: https://github.com/LedgerHQ/btchip-python/blob/c2bb2166cdaec95795517811c6f6e16038401f1d/btchip/btchipFirmwareWizard.py#L20-L21

Reply:
In the event that btchip implements the `checkFirmware` function, I'm assuming that they would take the firmware value in the same format that btchip-python returns. Do you think that I should revert it?